src/codeenginestudio/thuy/Exercise3.java

```diff
@@ -0,0 +1,68 @@
+package codeenginestudio.thuy;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class Exercise3 {
+    public static void main(String[] args) {
+        int n = 5;
+        int[][] ouput = invertMiddleElement(n);
+        System.out.println(Arrays.deepToString(ouput));
+    }
+
+    private static boolean checkMiddleElement(int number, int n) {
+        ArrayList middleElement = findMiddleElement(n);
+
+        if (middleElement.contains(number)) {
+            return true;
+        }
+        return false;
+    }
+
+    private static ArrayList findMiddleElement(int n) {
+        int[][] array = show2DArray(n);
+        ArrayList middleElementList = new ArrayList();
+
+        for (int i = 0; i < n; i++) {
+            for (int j = 0; j < n; j++) {
+
+                if (i == 0 || i == n - 1) {
+                } else if (j == 0 || j == n - 1) {
+                } else
+                    middleElementList.add(array[i][j]);
+            }
+        }
+        return middleElementList;
+    }
+
+    private static int[][] invertMiddleElement(int n) {
+        int[][] array = show2DArray(n);
+
+        for (int i = 0; i < n; i++) {
+            for (int j = 0; j < n; j++) {
+                if (checkMiddleElement(array[i][j], n) == true) {
```

Comment:
May be you can improve the logic in here. Ex: `checkMiddleElement(i, j)` and return `boolean`. 
No need `== true`, you can replace by `if (checkMiddleElement(array[i][j], n))`